src/schedule/zephyr_dp_schedule.c

```diff
@@ -227,11 +229,16 @@ void scheduler_dp_ll_tick(void *receiver_data, enum notify_id event_type, void *
 	lock_key = scheduler_dp_lock();
 	list_for_item(tlist, &dp_sch->tasks) {
 		curr_task = container_of(tlist, struct task, list);
+		pdata = curr_task->priv_data;
+		struct processing_module *mod = pdata->mod;
+
+		if (pdata->LL_cycles_to_deadline) {
```

Comment:
I would put an inline comment here to help with readability (since its efficient code block), e.g. to save all readers 30 secs in understanding.